contributed_definitions/NXellipsometry.nxdl.xml

```diff
@@ -305,35 +233,24 @@ A draft version of a NeXus application definition for ellipsometry.-->
                         </doc>
                     </field>
                 </group>
-                <group name="spectrometer" type="NXmonochromator" optional="true">
-                    <doc>
-                         The spectroscope element of the ellipsometer before the detector,
-                         but often integrated to form one closed unit. Information on the
-                         dispersive element can be specified in the subfield GRATING. Note
-                         that different gratings might be used for different wavelength
-                         ranges. The dispersion of the grating for each wavelength range can
-                         be stored in grating_dispersion.
-                    </doc>
-                </group>
             </group>
         </group>
         <group type="NXsample">
-            <field name="backside_roughness" type="NX_BOOLEAN">
+            <field name="backside_roughness" type="NX_BOOLEAN" recommended="true">
```

Comment:
Default value: wrong. Most often used reflective Si wafer samples do not need it.

Reply:
yea, right. Im used to the specific situation, that the substrate is mostly transparent an may therefore influnce the measurement. As it was required before, I now set it to optional.